example-mlem-get-started/generate.sh

```diff
@@ -0,0 +1,209 @@
+
+
+python train.py
+git add .mlem
+tick
+git commit -m "Train the model"
+git tag -a "2-train" -m "Model trained."
+
+
+python evaluate.py
+mlem apply rf iris.csv
+echo "sepal length (cm),sepal width (cm),petal length (cm),petal width (cm)
+0,1,2,3" > new_data.csv
+mlem apply rf new_data.csv -i --it pandas[csv]
+git add metrics.json
+tick
+git commit -m "Evaluate model"
+git tag -a "3-eval" -m "Metrics calculated"
+
+
+mlem init s3://example-mlem-get-started
+mlem clone rf s3://example-mlem-get-started/rf
+
+
+mlem declare builder pip pip_config -c target=build/ -c package_name=example_mlem_get_started
+git add .mlem
+tick
+git commit -m "Add package config"
+git tag -a "4-pack" -m "Pip package config added"
+
+mlem declare env heroku staging
+mlem declare deployment heroku myservice -c app_name=example-mlem-get-started-app -c model=rf -c env=staging
+git add .mlem
+tick
+git commit -m "Add env and deploy meta"
+git tag -a "5-deploy-meta" -m "Target env and deploy meta added"
+
+if heroku apps:info example-mlem-get-started-app; then
+  heroku apps:destroy example-mlem-get-started-app --confirm example-mlem-get-started-app
+fi
+
+mlem deployment run myservice
+git add .mlem
+tick
+git commit -m "Deploy service"
+git tag -a "6-deploy-create" -m "Deployment created"
+
+
+###################### DVC
+
+git checkout main
+git branch dvc
+git checkout dvc
+
+mlem init
+tick
+git add .mlem
+git commit -m "Initialize MLEM project"
+git tag -a "1-dvc-mlem-init" -m "MLEM initialized."
+
+dvc init
+dvc remote add myremote --local azure://example-mlem
+dvc remote add default -d https://examplemlem.blob.core.windows.net/example-mlem
+git add .dvc
+tick
+git commit -m "Init dvc"
+git tag -a "2-dvc-dvc-init" -m "DVC Initialized"
+
+
+mlem config set core.storage.type dvc
+echo "/**/?*.mlem" > .dvcignore
+git add .dvcignore .mlem
+tick
+git commit -m "Configure MLEM for DVC"
+git tag -a "3-dvc-mlme-config" -m "Configured MLEM to work with DVC"
+
+python train.py
+python evaluate.py
+dvc add .mlem/model/rf .mlem/data/*.csv
+git add .mlem .dvc metrics.json
+tick
+git commit -m "Run code with DVC"
+git tag -a "4-dvc-save-models" -m "Saved models with DVC storage"
+dvc push -r myremote
+
+popd
+
+unset TAG_TIME
+unset GIT_AUTHOR_DATE
+unset GIT_COMMITTER_DATE
+unset GIT_AUTHOR_NAME
+unset GIT_AUTHOR_EMAIL
+unset GIT_COMMITTER_NAME
+unset GIT_COMMITTER_EMAIL
+
+echo "`cat <<EOF-
+
+The Git repo generated by this script is intended to be published on
+https://github.com/iterative/example-mlem-get-started.
+Make sure the Github repo exists first and that you have
+appropriate write permissions.
+
+To create it with https://cli.github.com/, run:
+
+gh repo create iterative/example-mlem-get-started --public \
+     -d "Get Started MLEM project" -h "https://mlem.ai/doc/get-started"
+
+Run these commands to force push it:
+
+cd build/example-mlem-get-started
+git remote add origin  https://github.com/iterative/example-mlem-get-started
```

Comment:
extra space

+ let's rename `iterative` into `shcheklein` (or any other name) for example ... I found that people copy-paste and push this sometimes by mistake / too early